lib/bolt/outputter/human.rb

```diff
@@ -321,7 +321,7 @@ def print_plan_info(plan)
                 end
 
         plan['parameters'].each do |name, p|
-          pretty_params << "- #{name}: #{p['type']}\n"
+          pretty_params << "- #{name}: #{p['type'] || 'Any'}\n"
```

Comment:
I think puppet does this for you (i dont think it is possible to not have a type here?)

Reply:
Nice - this was intended for YAML plans, not Puppet plans, but it looks like it gets set there too :+1: